katnip/legos/url.py

```diff
@@ -265,6 +277,8 @@ def __init__(self, path=None, path_delim='/', fuzz_delims=True, fuzzable=True, n
         if path is not None:
             fields.append(Delimiter(name='main delim', value='/', fuzzable=fuzz_delims))
             path_parts = path.split(path_delim)
+            if not path_parts[0]:
```

Comment:
better to do path_delim.strip() on the previous line

Reply:
If `path="/index"` then
`path_parts = ["", "index"]`

If `path="index"` then
`path_parts = ["index"]`

This code: `if not path_parts[0]:` need for 1 variant